backend/database/models/tag.py

```diff
@@ -3,4 +3,7 @@
 
 class Tag(db.Model):
     id = db.Column(db.Integer, primary_key=True)
+    incident_id = db.Column(
```

Comment:
This one could probably be a many to many... Thoughts?

Reply:
@dwreeves? @jcockhren?

Reply:
I'm not sure. I feel like this should be an array of strings and not even be a table. This is how the Berkeley Copwatch data model had it, though. They had more attributes in their tags table too, I think.